docs/DeveloperGuide.md

```diff
@@ -270,91 +293,48 @@ The following sequence diagram shows how deleting a session works:
 
 It shares the same design considerations as what is mentioned in Add Session Feature.
 
+### Calculating Monthly Fee Feature
+The monthly `fee` feature allows user to quickly calculate the amount of money they would have received
```

Comment:
Should receive/would receive.

Would have received may wrongly suggest to reader that only past sessions would be included

Reply:
O.o right~! Have updated accordingly. Thanks 👍🏻